### DISCUSSION TITLE
How do can i include the dl_session in the code?

### DISCUSSION THREAD

**Discussion by SerialSeller** [OwO-Network/DLX#145] [Q&A]
Hi, i have setup docker and python and i want to use the pro endpoint but every time i get 
Error: {'code': 401, 'message': 'No dl_session Found'} 
i extracted  dl_session from deepl website but where do i write it inside the code and how?
and sorry to bother with silly question but i'm newbie and learning.

**Comment by missuo** [comment]
https://ssa.sx/deeplx
